src/test/Makefile

```diff
@@ -201,6 +201,7 @@ PMEM2_TESTS = \
 	pmem2_config\
 	pmem2_source\
 	pmem2_source_alignment\
+	pmem2_source_numa\
```

Comment:
this should be under `ifeq ($(OS_DIMM),ndctl)` below, right next to `pmem2_badblock_mocks`